engines/grim/font.cpp

```diff
@@ -175,7 +177,55 @@ void BitmapFont::load(const Common::String &filename, Common::SeekableReadStream
 	g_driver->createFont(this);
 }
 
-uint16 BitmapFont::getCharIndex(uint16 c) const {
+void BitmapFont::loadTGA(const Common::String &filename, Common::SeekableReadStream *index, Common::SeekableReadStream *image) {
+  	Image::TGADecoder dec;
+	bool success = dec.loadStream(*image);
+
+	if (!success)
+		return;
+
+	const Graphics::Surface *surf = dec.getSurface();
+
+	_filename = filename;
+	_numChars = index->size() / 10;
```

Comment:
macro for 10?

Reply:
It's a size of index entry. Added a const